addons/binding/org.openhab.binding.autelis/README.md

```diff
@@ -32,61 +34,61 @@ All devices support some of the following channels:
 
 | Channel Type ID         | Item Type    | Description  |
 |-------------------------|------------------------|--------------|----------------- |------------- |
```

Comment:
While you are one it... you would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍